syntax = "proto2";
option java_package = "io.tagalyzer.posts";

package tagalyzer;

service Tagalyzer {
  rpc CreatePost (PostRequest) returns (PostReply) {}
}

message PostRequest {
  optional string client_id = 1;
  optional string text = 2;
  optional string created_at = 3;
}

message PostReply {
  optional string id = 1;
}
